tutorials/Land/Soil/Water/equilibrium_test.jl

```diff
@@ -270,20 +269,28 @@ plot(
     title = "Equilibrium test",
 );
 plot!(
-    dons_arr[4]["soil.water.ϑ_l"],
-    dons_arr[4]["z"],
-    label = string("t = ", string(t[4]), "days"),
-);
-plot!(
-    dons_arr[6]["soil.water.ϑ_l"],
-    dons_arr[6]["z"],
-    label = string("t = ", string(t[6]), "days"),
+    dons_arr[2]["soil.water.ϑ_l"],
+    dons_arr[2]["z"],
+    label = string("t = ", string(t[2]), "days"),
 );
 plot!(
-    (dons_arr[1]["z"] .+ 10.0) .* slope .+ dons_arr[6]["soil.water.ϑ_l"][1],
-    dons_arr[1]["z"],
-    label = "expected",
+    dons_arr[7]["soil.water.ϑ_l"],
+    dons_arr[7]["z"],
+    label = string("t = ", string(t[7]), "days"),
 );
+function expected(z, z_interface)
+    ν = 0.495
+    S_s = 1e-3
+    α = 2.6
+    n = 2.0
+    m = 0.5
+    if z < z_interface
+        return -S_s * (z - z_interface) + ν
+    else
+        return ν * (1 + (α * (z - z_interface))^n)^(-m)
```

Comment:
@gagnelandmanna this is for the unsaturated portion. i had to numerically solve for the interface location offline though (wolfram alpha again 👀 )